google/cloud/storage/blob.py

```diff
@@ -339,22 +339,27 @@ def public_url(self):
         )
 
     @classmethod
-    def from_string(cls, uri, client=None):
+    def from_string(cls, uri, client=None, encryption_key=None):
         """Get a constructor for blob object by URI.
 
          :type uri: str
```

Comment:
Thanks for catching this. For the doc string, the indentation should be left-indented instead and aligned with the triple quotes. Please help fix, thanks!